Package.swift

```diff
@@ -14,13 +14,14 @@ let package = Package(
     ],
     dependencies: [
         .package(name: "MapboxCoreMaps", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.0.0-rc.1")),
+        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("14.0.1")),
```

Comment:
@SiarheiFedartsou FYI, MapboxNavigationNative will need to assume this version of MapboxCommon for the duration of v10.0.0-rc.1’s usage in the navigation SDK.

/cc @MaximAlien

Reply:
We're abandoning this in favor of a new release (v14.2.0) of Common.